apps/workers/openaiWorker.ts

```diff
@@ -71,34 +77,36 @@ export class OpenAiWorker {
   }
 }
 
-const IMAGE_PROMPT_BASE = `
-I'm building a read-it-later app and I need your help with automatic tagging.
-Please analyze the attached image and suggest relevant tags that describe its key themes, topics, and main ideas.
+function promptFactory(type: "text" | "web" | "pdf" | "image") {
+  const typeContent = {
+    text: "User Note",
+    web: "HTML page",
+    pdf: "PDF file",
+    image: "Image",
+  };
+  return `I'm building a read-it-later app and I need your help with automatic tagging.
+${
+  type === "web" || type === "pdf" || type === "text"
+    ? `You are currently analyzing the content of a ${typeContent[type]}, please analyze the content after the sentence "CONTENT START HERE:"`
+    : `Please analyze the attached image`
+}
+Suggest relevant tags that describe its key themes, topics, and main ideas.
```

Comment:
Do you mind if we move the prompt changes to another PR? I'm trying to be very careful with prompt changes as they can have unintended behaviors and they require some testing. For example, here we changed everything to `5-8 tags` and some people were complaining that the `3-5` that I had before were already too much.

I think for the sake of this PR let's just use the TEXT prompt without any changes. It'll probably be good enough :)